Readme.md

```diff
@@ -16,9 +16,7 @@ in this repository at [`docs`](https://github.com/timescale/timescaledb-toolkit/
 
 ## 🖥 Try It Out ##
 
-The extension comes pre-installed on all [Timescale Forge](https://console.forge.timescale.com/) instances, and also on our full-featured [`timescale/timescaledb-ha` docker image](https://hub.docker.com/r/timescale/timescaledb-ha).
-
-We also provide nightly builds as a docker images in `timescaledev/timescaledb-toolkit:nightly`.
+The extension comes pre-installed on all [Timescale Cloud](https://console.cloud.timescale.com/) instances, and also on our full-featured [`timescale/timescaledb-ha` docker image](https://hub.docker.com/r/timescale/timescaledb-ha).
```

Comment:
Since we're touching these lines, there technically shouldn't be a comma in the sentence.
```suggestion
The extension comes pre-installed on all [Timescale Cloud](https://console.cloud.timescale.com/) instances and also on our full-featured [`timescale/timescaledb-ha` docker image](https://hub.docker.com/r/timescale/timescaledb-ha).
```